### PR TITLE
KAA-1557: C SDK documentation issue

### PR DESCRIPTION
**Pull request by mlysenko**
Added a requirement to install cmoka to C SDK docs.

### CODE REVIEW

**Comment by rasendubi** [comment]
Do we need Maven to build SDK with tests?

**Comment by mlysenko** [comment]
As a client team, we don't. This change was requested by the server team. They use maven to build the whole Kaa (server, clients, etc.). The maven building failed without cmoka installed so @vadimol agreed to add it.

**Comment by rasendubi** [comment]
I mean we can disable testing in the maven build, so cmocka won't be needed.

**Comment by vadimol** [comment]
We can do so by passing -DskipTests. We compile and run tests by default

**Comment by rasendubi** [comment]
ok

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md [hunk 1]
```diff
@@ -35,6 +35,13 @@ To install dependencies on Ubuntu execute the following command:
 sudo apt-get install cmake build-essential
 ```
 
+To build C SDK with Maven or to run unit tests, the cmoka unit testing framework has to be install as well:
```
Comment (rasendubi) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md]: s/cmoka/cmocka
s/install/installed

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md [hunk 2]
```diff
@@ -35,6 +35,13 @@ To install dependencies on Ubuntu execute the following command:
...
+
+```
+sudo apt-get install libcmocka0 libcmoka-dev
```
Comment (rasendubi) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md]: s/libcmoka/libcmocka

**Review by rasendubi — CHANGES_REQUESTED** [review]

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md (hunk 2 above)
Comment (mlysenko) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/index.md]: My bad, thanks

**Review by rasendubi — APPROVED** [review]

**Pull request merged**